packages/services/src/services/Translation/Translation.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import root from 'window-or-global';
+
+/**
+ * @constant {string | string} Host for the Translation API call
+ * @private
+ */
+const _host = process.env.TRANSLATION_HOST || 'https://www.ibm.com';
+
+/**
+ * @constant {string | string} CORS proxy for lower environment calls
+ * @private
+ */
+const _proxy = process.env.CORS_PROXY || '';
+
+/**
+ * Translation API endpoint
+ *
+ * @type {string}
+ * @private
+ */
+const _endpoint = `${_host}/common/v18/js/data/jsononly`;
+
+/**
+ * Translation API class with methods for fetching i18n data for ibm.com
+ */
+class TranslationAPI {
+  /**
+   * Returns translation i18n data
+   *
+   * @returns {Promise<any>} Translation data
+   * @example
+   * import { TranslationAPI } from '@carbon/ibmdotcom-services';
+   *
+   * async function getTranslation() {
+   *   const response = await TranslationAPI.getTranslation();
+   *   return response;
+   * }
+   */
+  static async getTranslation() {
+    const lc = 'en'; // TODO: create utility for fetching lc
+    const cc = 'us'; // TODO: create utility for fetching cc
+    const currenthost = `${root.location.protocol}//${root.location.host}`;
+    const proxy = currenthost !== _host ? _proxy : '';
+    const url = `${proxy}${_endpoint}/${cc}${lc}.json`;
+
+    return await axios
+      .get(url, {
+        headers: {
+          'Content-Type': 'text/plain',
+        },
+      })
+      .then(response => response.data);
```

Comment:
Drupal is going to take over this service for us right? Is it too early to think about whether we keep the data structure, or if it requires a more relevant data model? If we know or want to be opinionated about that model this service would be good to do that remodel here. As an immediate example, we might want to combine the separated `socialSharing`, `footerThin` into `footerMenu`. We could also sync up our language here too. Either change the components to match the service API call or vice versa. Just some thoughts as we move forward.